modules/chamber/Makefile

```diff
@@ -2,8 +2,28 @@ CURL ?= curl
 CHAMBER_VERSION ?= 2.0.0
 CHAMBER ?= chamber
 
+ifdef AWS_VAULT
+	CHAMBER_AUTH = ""
+else
+	CHAMBER_AUTH = $(WITH_AWS)
+endif
+
+WITH_CHAMBER = $(CHAMBER_AUTH) chamber exec $(CHAMBER_SERVICES) --
+CHAMBER_EXPORT = $(CHAMBER_AUTH) chamber export -f dotenv $(CHAMBER_SERVICES) | xargs -I{} echo " {}" | tr '\n' ' '
```

Comment:
Can you explain what this command is doing?

Reply:
I don't think this env is needed any more because `INJECT_CHAMBER` is not used anywhere.

Reply:
https://github.com/sportech-dev/local-dev/blob/chamber-support/Makefile#L14

After that make process got access to all secrets from chamber as usual environment variables